lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -21,7 +21,9 @@
 from django.test import RequestFactory
 from edxmako.shortcuts import render_to_response  # pylint: disable=import-error
```

Comment:
I didn't add those -- just left them in place, but I am glad to remove -- should I?